test/unit/helpers-test.js

```diff
@@ -16,16 +16,5 @@ describe('helpers', function() {
         assert.equal(tester[0], 'hello');
         assert.equal(tester[1], 'hi');
     });
-
-    it('log will not thow if console is cleared', function() {
-        var tmpConsole = window.console;
-
-        window.console = null;
-        // this should not break
-        utils.log('testing');
-
-        // restore actual window console
-        window.console = tmpConsole;
-    });
```

Comment:
I removed this not because it was logging to the console, it's because it's not doing anything. Before this even runs, `utils.log` is already being set to `console.log`, so it doesn't matter if `console.log` is `null`, it'll still log to the console. This was relevant before you refactored the`utils.log` to simply be a passthrough.